libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway.hs

```diff
@@ -511,3 +514,58 @@ instance IsConwayUniv fn => ExecSpecRule fn "RATIFY" Conway where
     first (\case {})
       . computationResultToEither
       $ Agda.ratifyStep env st sig
+
+data ConwayGovCertExecContext era = ConwayGovCertExecContext
```

Comment:
@Soupstraw is it okay to reuse `ConwayCertExecContext` here without making a new type?

Reply:
Yeah, that's fine.